index.d.ts

```diff
@@ -22,6 +29,7 @@ declare namespace passport_openidconnect {
                                      profile?: OICProfile,
                                      accessToken?: string,
                                      refreshToken?: string,
+                                     params?: any,
```

Comment:
this variable name is quite unspecific. What params?

Reply:
Also: it'd be very advantageous to have some typing here. If you simply need it to be a string->string dictionary, we can make a type for that. It probably shouldn't truly be an `any`. Can you describe what it's for?

Reply:
no idea which `params` these are 😓 
it's provided by `Oauth2.prototype.getOAuthAccessToken` method, which is a separate lib. internally, they call `results`, which also doesn't help...
the only 2 properties I see we use in the code are `id_token` and `expires_in`. I can log it to check what else is in there and we can build it from there if you feel like that's beneficial. I'd prefer to extend whatever type definition oauth itself has, but guess what they got... https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/oauth/index.d.ts#L24